### ISSUE TITLE
Cold intolerance

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10514]
**Preferred term label:**
Cold intolerance

**Synonyms**
feeling cold when other people are not cold

**Definition (free text, please give PubMed ID)**
The opposite of https://hpo.jax.org/app/browse/term/HP:0002046

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hypothyroidism

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
HP:6000855
Cold intolerance
